ivy_tests/test_ivy/test_stateful/test_layers.py

```diff
@@ -1126,7 +1126,44 @@ def test_maxpool2d_layer(
         on_device=on_device,
     )
 
-
+    
+# MaxPool3D
+@handle_method(
+    method_tree="MaxPool3D.__call__",
+    x_k_s_p=helpers.arrays_for_pooling(min_dims=5, max_dims=5, min_side=1, max_side=4),
```

Comment:
The test looks good at first glance. If the errors are related to the array sizes, maybe you could try increasing the `max_side`.

Reply:
The reason why I increased the max_side parameter to 8 is, to test the MaxPool3d layer with larger input tensor sizes, as the larger input tensor sizes can result in different behavior or errors that might not be captured with smaller input sizes. Additionally, larger input sizes can help to identify any performance issues or bottlenecks in the implementation.